pages/common/make.md

```diff
@@ -31,3 +31,7 @@
 - Override variables defined in the Makefile by the environment:
 
 `make --environment-overrides {{target}}`
+
+- Override a variable defined in the Makefile via the command line:
+
+`make {{target}} {{variable}}={{override}}`
```

Comment:
```suggestion
- Override a variable defined in the Makefile:

`make {{target}} {{variable}}={{new_value}}`
```
I think `via the command line` is a little redundant. I assume you added it for contrast to the previous example. However, I think we could move this one before the `--environment-overrides` anyways, because it's less specific.